src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,54 @@
+"""Convert SSSOM to ROBOT template."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+from sssom import MappingSetDataFrame
+from sssom.parsers import parse_sssom_table
+
+ROBOT_ROW = {
+    'subject_id': 'ID',
+    'subject_label': '',
+    'object_id': 'A oboInOwl:hasDbXref',
+    'object_label': '',
+    'equivalence': '>A oboInOwl:source',
+    'ORCID': '>A oboInOwl:source SPLIT=|',
+}
+
+
+def sssom_to_robot_template(inpath: Union[str, Path], outpath: Union[str, Path]):
+    """Convert SSSOM to ROBOT template."""
+    msdf: MappingSetDataFrame = parse_sssom_table(inpath)
```

Comment:
## General thoughts: `parse_sssom_table()`
@matentzn We had a recent discussion that it would be nice to have `msdf.to_csv()`. It'd also be nice to have `read_csv()` or `read_tsv()`. I feel like this is a better name for what this function does, and also consistent w/ pandas, improving usability.

Reply:
I hear you but this comment is not actionable in this PR; this would have to be implemented in sssom-py

Reply:
NP; I guess your preference is not to bring it up in the PR and just make an issue. I just like to bring it up in advance of making the issue, but I guess going straight to the issue is fine; I'll try to remember this preference, as it is a habit of mine to bring up tangential thoughts in PRs.

Reply:
I changed this issue from being about allowing `MappingSetDataFrame` to accept paths, to instead being about these `pandas`-like functions/methods:
- https://github.com/mapping-commons/sssom-py/issues/517